enterprise/server/cmd/goinit/main.go

```diff
@@ -312,6 +312,7 @@ func main() {
 	}
 	die(os.WriteFile("/etc/hosts", []byte(strings.Join(hosts, "\n")), 0755))
 	die(os.WriteFile("/etc/resolv.conf", []byte("nameserver 8.8.8.8"), 0755))
+	die(syscall.Symlink("../proc/self/mounts", "/etc/mtab"))
```

Comment:
Note, this was needed because `gosigar` was returning an error that `/etc/mtab` doesn't exist. On my system, this file is just a symlink to `../proc/self/mounts` so creating that symlink seemed to do the trick.

Reply:
Yep, done. Also it looks like this can be `/proc/mounts` instead of `/proc/self/mounts`.